controller/setting_controller.go

```diff
@@ -524,22 +534,71 @@ func getFinalTolerations(existingTolerations, lastAppliedTolerations, newTolerat
 }
 
 func (bm *BackupStoreMonitor) Start() {
+	log := bm.logger.WithFields(logrus.Fields{
```

Comment:
Add controller ID

Reply:
We already have a `node` field as part of the initial logger and we use a component field for the backup-store-monitor.
Not sure what adding a controllerID field would get us?